tests/test_geom_opt.py

```diff
@@ -21,13 +17,17 @@
 test_data = [
     ("mace", "NaCl.cif", -27.046359959669214, {}),
     ("mace", "NaCl.cif", -27.04636199814088, {"fmax": 0.001}),
-    ("mace", "NaCl.cif", -27.0463392211678, {"filter_func": UnitCellFilter}),
+    ("mace", "NaCl.cif", -27.0463392211678, {"filter_func": FrechetCellFilter}),
```

Comment:
I don't think we want this, as it's the default, so this doesn't really check it can be set?

Reply:
do not we have 3.23 as minimum version? so we will not really have to check it since is there.

Reply:
It's not checking the availability of the functions from ASE, it was doing a very rough check that `filter_func` can be set as a different function in general.